client/flutter/lib/pages/settings_page.dart

```diff
@@ -127,6 +133,45 @@ class _SettingsPageState extends State<SettingsPage>
                         .homePagePageSliverListSettingsNotificationsInfo,
                     isToggled: _notificationsEnabled ?? false,
                     onToggle: _toggleNotifications),
+                SizedBox(
+                  height: 20,
+                ),
+                divider,
+                menuItem(
+                  title: S.of(context).homePagePageSliverListShareTheApp,
+                  onTap: () {
+                    FirebaseAnalytics().logShare(
+                        contentType: 'App',
+                        itemId: null,
+                        method: 'Website link');
+                    Share.share(
+                      S.of(context).commonWhoAppShareIconButtonDescription,
+                      sharePositionOrigin:
+                          Rect.fromLTWH(0, 0, size.width, size.height / 2),
+                    );
+                  },
+                ),
+                divider,
+                // TODO: Localize
+                menuItem(
+                    title: 'Provide app feedback',
+                    onTap: () {
+                      FirebaseAnalytics().logEvent(name: 'Feedback');
+                      // TODO: Implement feedback #989 #1015
+                    }),
+                divider,
+                menuItem(
+                  title: S.of(context).homePagePageSliverListAboutTheApp,
+                  onTap: () {
+                    FirebaseAnalytics().logEvent(name: 'About');
+                    return Navigator.of(context)
+                        .push(MaterialPageRoute(builder: (c) => AboutPage()));
+                  },
+                ),
+                divider,
+                Container(
+                  height: 25,
```

Comment:
The nitpickiest nit that ever did pick: David requested all gutters be sizes in multiples of 4 until we have a more coherent way of handling padding. Not necessary if you don't want to, but:

```suggestion
                  height: 24,
```

Reply:
Done.  However David's paddings are not currently multiples of 4 in his figmas, so won't touch the others.